ridgeback_control/launch/teleop.launch

```diff
@@ -20,8 +20,11 @@
     </node>
   </group>
 
+  <arg name="config" value="planar" />
 
   <node pkg="interactive_marker_twist_server" type="marker_server" name="twist_marker_server">
     <remap from="twist_marker_server/cmd_vel" to="/cmd_vel" />
+    <rosparam command="load" file="$(find interactive_marker_twist_server)/config/$(arg config).yaml" />
```

Comment:
Might want to use `config/teleop_$(arg config).yaml` since this package has lots of different config files in it.